src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -680,13 +721,12 @@ def create_zonegroup(self, realm_name: str, zonegroup_name: str,
             cmd_create_zonegroup_options.append(endpoint)
         rgw_zonegroup_create_cmd += cmd_create_zonegroup_options
         try:
-            exit_code, out, _ = mgr.send_rgwadmin_command(rgw_zonegroup_create_cmd)
+            exit_code, out, err = mgr.send_rgwadmin_command(rgw_zonegroup_create_cmd)
             if exit_code > 0:
-                raise DashboardException('Unable to get realm info',
+                raise DashboardException(e=err, msg='Unable to get realm info',
```

Comment:
Are we displaying both the error and the msg?

Reply:
Yes